command/config/config.go

```diff
@@ -390,7 +390,8 @@ func FromEnviron() (EnvConfig, error) {
 		config.Runner.Environ = map[string]string{}
 	}
 	if config.Runner.Name == "" {
-		config.Runner.Name, _ = os.Hostname()
+		hostname, _ := os.Hostname()
+		config.Runner.Name = fmt.Sprintf("runner-%s", hostname)
```

Comment:
Add this only if host name is empty string or starts with non lower case alphabet

Reply:
Done

Reply:
Should we do `strings.ToLower(hostname)` by default since Google API has a hard check on that?

Regex in the error: `(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)`